features/step_definitions/kata.rb

```diff
@@ -227,3 +227,78 @@
     end
   end
 end
+
+Given /^Verify catalog source existence$/ do
+  project_name = 'openshift-marketplace'
+  catalog_source_name = 'redhat-operators'
+  step %Q/I switch to cluster admin pseudo user/
+  project(project_name)
+  if !catalog_source(catalog_source_name).exists?
+    logger.warn("Missing #{catalog_source_name} catalog source")
+    logger.info("Creating #{catalog_source_name}")
+    step %Q/I create "redhat-operators" catalogsource for testing/
+    sleep(120)
+    raise "Failed to create catalog source" unless catalog_source('redhat-operators').exists?
+  elsif catalog_source('redhat-operators').exists?
+    logger.info("Catalog source #{catalog_source_name} exists")
+  end
+end
+
+Given /^Install Kata operator$/ do
```

Comment:
@valtair1891 your step definitions doesn't really follow the conventional cucumber steps.  If you take a look at some other tests, you will see that what you have written doesn't have a nice flow.   https://cucumber.io/docs/bdd/better-gherkin/

Reply:
@pruan-rht Are you just referring to the use of "and" instead of another "Given"?  Again, are we required to follow some coding standard?

Reply:
@cpmeadors from cucumber's POV, these keywords are interchangable.  But we do following naming convention as explained here https://cucumber.io/docs/gherkin/reference/

Reply:
Given /^I install Kata operator$/